src/code/z_actor.c

```diff
@@ -3509,16 +3509,16 @@ void func_80033F54(GlobalContext* globalCtx, s32 arg1, s32 arg2) {
     f32 temp2;
     f32 temp3;
 
-    entry = &D_801160DC[arg2];
+    entry = &D_801160DC[type];
     var = entry->unk_10;
 
     OPEN_DISPS(globalCtx->state.gfxCtx, "../z_actor.c", 8265);
 
     Matrix_Translate(0.0f, entry->unk_08, 500.0f, MTXMODE_APPLY);
     Matrix_Get(&spB0);
 
-    temp1 = sinf(entry->unk_00 - var) * -(10 - arg1) * 0.1f * entry->unk_04;
-    temp2 = cosf(entry->unk_00 - var) * (10 - arg1) * 0.1f * entry->unk_04;
+    temp1 = sinf(entry->unk_00 - var) * -(10 - frame) * 0.1f * entry->unk_04;
+    temp2 = cosf(entry->unk_00 - var) * (10 - frame) * 0.1f * entry->unk_04;
```

Comment:
what does frame mean here? frame count in the opening animation or something? I havent read the code its called from yet, but it may be worth being more specific anyway

Reply:
It's the animation frame, yes. If more clarity is desired I could change it to `animFrame`?

Reply:
i would personally, but its probably pretty clear from context too. whatever you feel is best